specifyweb/frontend/js_src/lib/components/Formatters/Definitions.tsx

```diff
@@ -197,12 +197,14 @@ function ConditionalFormatter({
           {resourcesText.addField()}
         </Button.Small>
       ) : (
-        fields.map((field, index) => (
-          <p key={index}>
-            {field.separator === undefined ? '' : field.separator}
-            {field.field === undefined ? '' : field.field[0].label}
-          </p>
-        ))
+        <div className="flex flex-wrap p-2">
+          {fields.map((field, index) => (
+            <p key={index}>
```

Comment:
why is this P even needed? can you use React.Fragment instead?
this p is causing too much trouble 🤣

Reply:
and these are just strings anyway, right?
so you could just use do `.map().join()`, so no need for React.Fragment at all